src/ctia/encryption/default.clj

```diff
@@ -15,17 +17,18 @@
               slurp
               string/trim)))
 
-(defrecord EncryptionService [state]
+(tk/defservice default-encryption-service
```

Comment:
In IROH we try to declare the protocol along the defservice.
I think it would be nice to merge the two ns in another PR.

Regarding the name of the service, I think I would name it `encryption-service` and distinct between other service implementing the same protocol via the namespace.

Reply:
Thanks, I'll make a note. I think it'll be easy to make a sweep to use standard TK conventions after the dust has settled .